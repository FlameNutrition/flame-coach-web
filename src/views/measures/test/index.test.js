import React from 'react';
import MeasuresView from '../index';
import { render } from '../../../testing/test-utils';

jest.mock('../../../components/Charts/WeightChart',
  () => () => '-- PLACEHOLDER WeightChart --');

jest.mock('../../../components/Charts/Events',
  () => () => '-- PLACEHOLDER Events --');

jest.mock('../../../components/Charts/Filters',
  () => () => '-- PLACEHOLDER Filters --');

describe('<MeasuresView />', () => {
  it('create component MeasuresView', () => {
    const { container } = render(<MeasuresView />);

    expect(container)
      .toMatchSnapshot();
  });
});
