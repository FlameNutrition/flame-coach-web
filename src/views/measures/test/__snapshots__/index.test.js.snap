// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MeasuresView /> create component MeasuresView 1`] = `
<div>
  <div
    class="makeStyles-root-1"
  >
    <div
      class="MuiContainer-root"
    />
  </div>
</div>
`;
